pkg/cmd/get/projects.go

```diff
@@ -33,31 +34,61 @@ var projectsCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		credentials, err := utils.GetCredentials(cmd)
 		utils.PrintError(err)
-		projects, err := apis.ListProject(credentials)
-		utils.PrintError(err)
 
-		output, err := cmd.Flags().GetString("output")
+		//promptui to ask the user for the output format
+		outputFormat := ""
+
+		projects, err := apis.ListProject(credentials)
 		utils.PrintError(err)
 
-		switch output {
-		case "json":
+		switch outputFormat {
+		case "JSON":
```

Comment:
string constants can follow smallcase as mentioned earlier

Reply:
I updated all of the above like removing the commented code and fixing casing in the recent commit.